docs/src/rules/no-undef.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: no-undef
 rule_type: problem
+handled_by_typescript:
+- 2304
```

Comment:
`2552`: _`Cannot find name 'someFunction'. Did you mean 'Function'?`_

https://www.typescriptlang.org/play?#code/G4QwTgBAZg9jEF4IGcYFsCmAxArgOwGMAXASxjwAoBKAbgChRIAjcRCECAaggEYag